Controller/Adminhtml/Bread/GenerateCart.php

```diff
@@ -104,7 +103,7 @@ public function execute()
 
             $ret['cartUrl'] = $result['url'];
             $ret['id'] = $result['id'];
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```

Comment:
in PHP it looks like there's Exceptions and Errors, which both implement Throwable, so any Errors that were thrown were not being caught, so I figured since they were generic catch block, we should change it to Throwable to gracefully catch anything that goes wrong, not just Exceptions